rules_python_gapic/py_gapic_pkg.bzl

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@com_google_api_codegen//rules_gapic:gapic_pkg.bzl", "construct_package_dir_paths")
+
+def _py_gapic_src_pkg_impl(ctx):
+    srcjar_srcs = []
+    for dep in ctx.attr.deps:
+        for f in dep.files.to_list():
+            if f.extension in ("srcjar", "jar", "zip"):
+                srcjar_srcs.append(f)
```

Comment:
Does skylark allow list comprehensions? Personal preference, feel free to ignore, but I think
```python
srcjar_srcs = [
    dep_file
    for dep in ctx.attr.deps
    for dep_file in dep.files.to_list()
    if dep_file.extension in ("srcjar", "jar", "zip")
]
```
is more readable because of the indentation.

Reply:
It is a reduced verison of more complicated for loops copied from the other rules. Starlak support list comprehension. For sake of style conssstency with the other rules lets keep it like that.